eth/vm/gas_meter.py

```diff
@@ -70,14 +103,4 @@ def return_gas(self, amount: int) -> None:
         )
 
     def refund_gas(self, amount: int) -> None:
-        if amount < 0:
-            raise ValidationError("Gas refund amount must be positive")
-
-        self.gas_refunded += amount
-
-        self.logger.trace(
-            'GAS REFUND: %s + %s -> %s',
-            self.gas_refunded - amount,
-            amount,
-            self.gas_refunded,
-        )
+        return self.refund_strategy(self, amount)
```

Comment:
Then this would be something like:
```suggestion
        self.gas_refunded = refund_strategy(self.gas_refunded, amount)
        
        self.logger.trace(	
            'GAS REFUND: %s + %s -> %s',	
            self.gas_refunded - amount,	
            amount,	
            self.gas_refunded,	
        )

        return self.gas_refunded
```